_topics/en/transaction-bloom-filtering.md

```diff
@@ -50,6 +50,9 @@ optech_mentions:
   - title: "Bitcoin Core #19260 disconnects peers who inappropriately send filterclear"
     url: /en/newsletters/2020/06/24/#bitcoin-core-19260
 
+  - title: "History of the `mempool` P2P message deprecated along with BIP37 bloom filters"
```

Comment:
Is the following too long:

```suggestion
  - title: "History of the `mempool` P2P message being deprecated along with BIP37 bloom filters"
```

(otherwise "deprecated" modifies "History of.." rather than "mempool P2P message")

Reply:
Ive added this word and was able to build without any error or visible issue.